src/runtime/relax_vm/executable.cc

```diff
@@ -75,6 +75,10 @@ PackedFunc Executable::GetFunction(const String& name, const ObjectPtr<Object>&
       vm->LoadExecutable(GetObjectPtr<Executable>(this));
       *rv = Module(vm);
     });
+  } else if (name == "func_exists") {
```

Comment:
thinking a bit more, how about rename to has_function

Reply:
sounds good. I'm updating the name on my end to accelerate PR merge